scripts/population/build-cpp

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+# TODO: get app folder from env
```

Comment:
So this script only works from docker?
BASH_SOURCE and dirname can be used:

Maybe:

`cd "$( dirname "${BASH_SOURCE[0]}" )/../../cpp"`